Server/Routes/entryRoutes.js

```diff
@@ -4,12 +4,14 @@ import specificentry from '../Controller/getspecificentry';
 import addentry from '../Controller/addentry';
 import modifyentry from '../Controller/modifyentry';
 import deleteentry from '../Controller/deleteentry';
+import validator from '../validation/validation'
 
 const router = express.Router();
 
 
 router.get('/api/v1/entries', getallentriesl);  
 router.get('/api/v1/entries/:id', specificentry);
+ router.post('/api/v1/entries/', validator.validateString, addentry);
```

Comment:
You didn't change anything from the code I wrote. Now you have 2 posts on the same route. Please edit this code to match what you want it to do.